code/modules/background/accent/tajara.dm

```diff
@@ -34,30 +34,30 @@
 /datum/accent/high_harr_siik
 	name = ACCENT_HIGHHARRSIIK
 	description = "Used by the natives of northern Harr'masir. It is deemed to be a rustic and harsh-sounding form of siik'maas; the accent is associated with peasants and uneducated \
-	Tajara. It is the most common dialect among New Kingdom's commoners. The nobility usually attempts to mask this accent due to its relationship with the common people. It is also \
+	Tajara. It is the most common dialect among the New Kingdom's commoners. The nobility usually attempts to mask this accent due to its relationship with the common people. It is also \
```

Comment:
```suggestion
	Tajara. It is the most common dialect among New Kingdom's commoners. The nobility usually attempts to mask this accent due to its relationship with the common people. It is also \
```
previous seems correct, since it's a proper noun.

Reply:
But it’s a descriptive name. You wouldn’t say “the Germany’s” but you would say “the United Kingdom’s.”